content/collections/extending-docs/addons.md

```diff
@@ -460,6 +460,57 @@ if ($addon->edition() === 'pro') {
 > You don't need to check whether a license is valid, Statamic will do that automatically for you.
 
 
+## Update Scripts
+
+You may register update scripts to help your users migrate data, etc. when new features are added, breaking changes are introduced, etc.
```

Comment:
Extraneous "etc" in the middle of the sentence

Reply:
Ah yes, thanks!